test/reduce_test.cpp

```diff
@@ -843,6 +825,14 @@ struct reduce_driver : test_driver
                     tensor_elem_gen_checkboard_sign{}(is...));
         };
 
+        if(reduceOp == MIOPEN_REDUCE_TENSOR_NORM1 || reduceOp == MIOPEN_REDUCE_TENSOR_NORM2)
+        {
+            if(toReduceDims.size() == 4)
+                this->tolerance = 80 * 500;
```

Comment:
[Q] Are you pretty sure that implementation is correct and increasing of tolerance is justified?

Reply:
Yes.  `std::numeric_limits<float>::epsilon()  * (80*500)`  is 0.00476837158203, not abnormally large

Reply:
Ok